src/Makefile.am

```diff
@@ -2109,10 +2109,10 @@ tests_testHttpReply_LDADD=\
 	$(top_builddir)/lib/libmisccontainers.la \
 	$(top_builddir)/lib/libmiscencoding.la \
 	$(top_builddir)/lib/libmiscutil.la \
-	$(NETTLELIB) \
 	$(SSLLIB) \
 	$(LIBCPPUNIT_LIBS) \
 	$(COMPAT_LIB) \
+	$(LIBNETTLE_LIBS) \
```

Comment:
Do we need to change the order of LDADD entries? If yes, perhaps the PR description should explain why? If this and similar change are accidental, then perhaps we should keep the official order in this PR? This is just a comment. I do not know enough to insist on any changes.

Reply:
Sure. Updated description.